packages/thirdweb/src/chainsaw/paging.ts

```diff
@@ -0,0 +1,14 @@
+import type { ChainsawPagingParams } from "./types.js";
+
+export const addPagingToRequest = (
```

Comment:
prefer `function foo` over `const foo () =>` wherever possible

Reply:
suggestion: `export function addRequestPagination(...`

Reply:
Done in [a9dd446](https://github.com/thirdweb-dev/js/pull/3741/commits/a9dd4468910552fb203852aa9711d7d358988339) and [0533dd2](https://github.com/thirdweb-dev/js/pull/3741/commits/0533dd2fe813e55615cb6b263504abc6462b49b9)

Just out of curiosity, is there a specific reason to prefer `function`?

Reply:
nvm, saw the `mostly style` comment